content/news/2020-11-13-release-plans-and-call-for-contrib.md

```diff
@@ -0,0 +1,48 @@
+type = "news"
+title = "R1 Status Update and Call for Contributors"
+author = "bitigchi"
+date = "2020-11-13 13:00:00+03:00"
+tags = ["r1"]
+
+After nearly 20 years of development, Haiku is very close to its first release. We need your help to make it happen. Here's a status update for R1, and latest project news.
+
+## Release Plans
+
+We are planning to release Beta3 sometime next year, hopefully followed by the first stable release, R1. We currently have [31](https://dev.haiku-os.org/query?status=assigned&status=in-progress&status=reopened&status=new&group=status&milestone=R1%2Fbeta3) tickets left for Beta 3, and [688](https://dev.haiku-os.org/query?status=assigned&status=in-progress&status=reopened&status=new&group=status&milestone=R1) tickets for R1.
+
+Haiku project is by no means a small project, it is a full operating system nonetheless. Maintaining a full-stack operating system is a huge work, and needs more than a handful of developers. Recently, one of our core developers, waddlesplash has recently announced his leave from the project due to personal reasons. waddlesplash has contributed to Haiku Project close to a decade, and has been the top contributor for 2017, 2018, and 2019. He has contributed invaluable additions to the Haiku Project, like the NVME driver, and USB3 support, with countless bug fixes and improvements on numerous Haiku components.
+
+**With this departure and the prolonged absence of the other core developers, currently we do not have the enough developer power to release R1. More beta releases are always possible, but without a stable release, Haiku will never have the chance to become a viable alternative for desktop users.**
+
+Making Haiku something more than an enthusiast operating system is not possible without your help.
```

Comment:
Do you plan to pay for people? Because if you don't, you are only attracting more people enthusiast enough to work for free. And that will make it a larger enthusiast operating system.

Reply:
This is not constructive criticism, it's sheer cynicism. Paying people is not the issue. Haiku project needs people motivated enough to put a release out. It's obvious that the current developers have lost their motivation to fix bugs, and frankly, that's okay. We just need to get fresh blood to put the project back on track.

Reply:
I don't see how else you can motivate people to fix these 688 bugs. Good luck.

Reply:
"more than an enthusiast operating system" is subjective. Please be more neutral, for instance: "Achieving a stable release of Haiku is not .."

Reply:
Done.